dojo/notifications/helper.py

```diff
@@ -72,7 +72,7 @@ def create_notification(event=None, *args, **kwargs):
                 applicable_notifications.append(system_notifications)
 
             notifications_set = Notifications.merge_notifications_list(applicable_notifications)
-
+            notifications_set.user = user
```

Comment:
I don't think the notifications_set.user can ever be None, but it doesn't hurt too bad to set it again